docs/developers/omnichain/tutorials/curve.md

```diff
@@ -27,7 +27,7 @@ it take a look to official script, does all the work for you out of the box
 [deployment script](https://github.com/curvefi/curve-contract/blob/master/scripts/deploy.py)),
 using the address of three ZRC-2020 tokens. You can find the ZetaChain addresses
 of ZRC-20 tokens supported right now on the Athens-2 testnet using this
-[endpoint](https://api.athens2.zetachain.com/#/Query/ZetachainZetacoreFungibleForeignCoinsAll).
+[endpoint](DOES NOT WORK https://api.athens2.zetachain.com/#/Query/ZetachainZetacoreFungibleForeignCoinsAll).
```

Comment:
This query doesn't work with a new athens3 endpoint like BlockPi or Ankur. I don't think it's included in Athens 3 at all. 

@lucas-janon @fadeev What should this be updated to?

Reply:
Thanks! Fixed: https://github.com/zeta-chain/docs/pull/121